litmus-portal/frontend/src/components/Sections/CreateWorkflow/ReliabilityScore/index.tsx

```diff
@@ -9,7 +9,7 @@ import * as WorkflowActions from '../../../../redux/actions/workflow';
 import { RootState } from '../../../../redux/reducers';
 import ButtonFilled from '../../../Button/ButtonFilled';
 import ButtonOutline from '../../../Button/ButtonOutline';
-import InfoTooltip from '../../../InfoTooltip';
+// import InfoTooltip from '../../../InfoTooltip';
```

Comment:
remove the commented code

Reply:
We need it in next release. St
ill should I?

Reply:
BCH will detect the commented codes. Currently, we don't have too much-commented code, that why it is not showing errors.
![Selection_025](https://user-images.githubusercontent.com/20414564/92711648-55d60000-f376-11ea-8173-9e34182d38dd.png)

Reply:
Done